src/mono/mono/mini/CMakeLists.txt

```diff
@@ -439,7 +439,7 @@ if(NOT DISABLE_SHARED_LIBS)
   endif()
 endif()
 
-if(HOST_WASM)
+if(HOST_WASM AND NOT HOST_WASI)
```

Comment:
Can both be defined at the same time?

Reply:
I thought so, yes - I thought `HOST_WASI` was like a subset of `HOST_WASM`. But I might be misremembering!